docs/1.1-DRAFT/index.md

```diff
@@ -377,14 +379,13 @@ The _RO-Crate Metadata File Descriptor_ MAY contain information such as
 licensing for the _RO-Crate Metadata File_ so metadata can be licensed
 separately from Data.
 
-
 The table below outlines the properties that the _Root Data Entity_ MUST have to be minimally valid and additionally highlights properties required to meet other common use-cases, including the minimum metadata necessary to mint a DataCite DOI:
 
 #### Direct properties of the Root Data Entity
 
 The _Root Data Entity_ MUST have the following properties:
 *  `@type`: MUST be [Dataset]
-*  `@id`:  MUST be a a string of ‘./’
+*  `@id`:  MUST end with `/` and SHOULD be the string `./`
```

Comment:
How can a parser know what is the _Root Data Entity_ in this case?

Reply:
They would need to follow the link from the [RO-Crate Metadata File Descriptor ](https://researchobject.github.io/ro-crate/1.0/#core-metadata-for-the-root-data-entity)

> 1. For each entity in `@graph` array
> 2. ..if the `conformsTo` property is a URI that starts with `https://w3id.org/ro/crate/`
> 3. ....from this entity's `about` object keep the `@id` URI as variable _root_
> 4. For each entity in `@graph` array
> 5. .. if the entity has an `@id` URI that matches _root_ return it

but I am also not too confident with this change - perhaps @markwilkinson can comment on if it is still needed to have absolute URIs as in <http://ldp.cbgp.upm.es:8890/DAV/coronavirus/ro-crates/GobiernoEspana/ro-crate-metadata.jsonld> ?

```jsonld
    {
      "@id": "ro-crate-metadata.jsonld",
      "@type": "CreativeWork",
      "about": {
        "@id": "http://w3id.org/FAIR_COVID19/DAV/coronavirus/ro-crates/GobiernoEspana"
      }
  {
      "@id": "http://w3id.org/FAIR_COVID19/DAV/coronavirus/ro-crates/GobiernoEspana",
      "@type": "Dataset",
      "name": [
        "Situación de COVID-19 en España"
      ]
}

Reply:
I think he is using my library, and that property is not included!

fbacall/ro-crate-ruby#12